File: Online or Offline.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-->0)
        {
            float a=sc.nextInt();
            float b=sc.nextInt();
            float c=a-(a/10);
            if(c<b)
            System.out.println("ONLINE");
            else if(c==b)
            System.out.println("EITHER");
            else
            System.out.println("DINING");
        }
    }
}

File: Online or Offline.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        float a,b;
        scanf("%f%f",&a,&b);
        float c=a-(a/10);
        if(c<b)
        printf("ONLINE\n");
        else if(c>b)
        printf("DINING\n");
        else
        printf("EITHER\n");
    }
}
